salt/kubelet/kubelet.jinja

```diff
@@ -32,5 +44,6 @@ KUBELET_ARGS="\
     --cloud-provider=aws \
 {% endif -%}
     {{ pillar['components']['kubelet']['args'] }} \
+    --node-labels={{ node_labels|join(',') }} \
```

Comment:
https://docs.saltstack.com/en/latest/ref/states/all/salt.states.k8s.html might work, assuming we have a new enough salt, and it doesn't have the same issue the ansible module did - no ability to use client certs.